.github/workflows/push.yml

```
@@ -14,13 +14,23 @@
+
+      # Build talawa api compose testing environment
+      - name: Build talawa api compose testing environment
+        run: docker compose build
+
+      # Run tests
+      - name: Run tests
+        run: docker compose up --exit-code-from api
+
+      # Copy coverage from container
+      - name: Copy coverage from container
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:11: Double quote to prevent globbing and word splitting

(shellcheck)